admin/load.php

```diff
@@ -291,5 +300,11 @@ function perflab_get_module_data( $module_file ) {
 		$module_data['experimental'] = false;
 	}
 
+	// Extract the module focus from its filename.
+	preg_match( '/.*\/modules\/([^\/]*)\/.*/', $module_file, $matches );
+	if ( isset( $matches[1] ) ) {
+		$module_data['focus'] = $matches[1];
+	}
```

Comment:
Further above, we already have a `dirname( $module_file )` which parses this into a simpler format of just `{focus}/{module-slug}`. It would be a bit simpler/more performant to assign that to a variable (e.g. `$module_dir`) instead and then parse the two parts using `explode`, e.g.:
```php
list( $focus, $slug ) = explode( '/', $module_dir );
```

Since now the array keys are the module dirs, I think it would be worth including the `slug` separately within `$module_data` as well.

Reply:
updated the approach to use `$module_dir` in [`06a5a78` (#58)](https://github.com/WordPress/performance/pull/58/commits/06a5a786bd327afd137655471b301ae0d4ed946d)

Reply:
added slug to module data (plus data validation) in [`1d4984d` (#58)](https://github.com/WordPress/performance/pull/58/commits/1d4984d4a31ca2b8e09512acb29fcf37d07b8bfb)